src/main/java/labs/introtoprogramming/lab5/object/Builder/POJO/FacePlate.java

```diff
@@ -0,0 +1,54 @@
+package labs.introtoprogramming.lab5.object.Builder.POJO;
+
+import java.util.ArrayList;
+
+public class FacePlate {
+
+    public ArrayList<VertexFace> vertices = new ArrayList<VertexFace>();
+    public Material material = null;
+    public Material map = null;
+
+    public FacePlate() {
```

Comment:
Add constructor for creating face with given vertices, material and map. And than make them private or at least package-private

Reply:
Done:)